### DISCUSSION TITLE
Schedule a function to run at a specific time with taskiq-faststream

### DISCUSSION THREAD

**Discussion by Atanusaha143** [ag2ai/faststream#1916] [Q&A]
Can I do something like below?

```python
@taskiq_broker.task(
    schedule=[{"cron": "* * * * *"}],  # Runs every minute
)
async def my_scheduled_task():
    print("do something")
```

Or any other way to run a function at a specific time?

NOTE: I don't want to use any broker

**Comment by Lancetnik** [comment]
You should check the **taskiq** documentation to watch all its' features and API. **FastStream** is just a publisher in this integration, no more
